### DISCUSSION TITLE
Hide Path C++ for VSCode

### DISCUSSION THREAD

**Discussion by tmnguyenri** [community/community#136088] [Other Feature Feedback, Questions, & Ideas]
Just want a clean output without this.
Hope people won't tell me use Code Runner again. :D

![image](https://github.com/user-attachments/assets/3a3bbaaa-4036-4092-9d86-8c1bcafbd5dd)

**Comment by thanhdatpb** [comment]
Accept

**Reply by tmnguyenri** [reply]
Có bí quyết gì không ạ?
